src/rules/html-closing-bracket-spacing.ts

```diff
@@ -0,0 +1,127 @@
+import { createRule } from "../utils"
+import type { AST } from "svelte-eslint-parser"
+
+export default createRule("html-closing-bracket-spacing", {
+  meta: {
+    docs: {
+      description: "Require or disallow a space before tag's closing brackets",
+      category: "Stylistic Issues",
+      conflictWithPrettier: true,
+      recommended: false,
+    },
+    schema: [
+      {
+        type: "object",
+        properties: {
+          startTag: {
+            enum: ["always", "never", "any"],
+          },
+          endTag: {
+            enum: ["always", "never", "any"],
+          },
+          selfClosingTag: {
+            enum: ["always", "never", "any"],
+          },
+        },
+        additionalProperties: false,
+      },
+    ],
+    messages: {
+      expectedSpace: "Expected space before '>', but not found.",
+      unexpectedSpace: "Expected no space before '>', but found.",
+    },
+    fixable: "code",
+    type: "layout",
+  },
+  create(ctx) {
+    const options = {
+      startTag: "never",
+      endTag: "never",
+      selfClosingTag: "always",
+      ...ctx.options[0],
+    }
+    const src = ctx.getSourceCode()
+
+    /**
+     * Returns true if string contains newline characters
+     */
+    function containsNewline(string: string): boolean {
+      return /.*\n.*/s.test(string)
+    }
+
+    /**
+     * Returns true if string contains whitespace characters
+     */
+    function containsWhitespace(string: string): boolean {
+      return /.*\s.*/s.test(string)
+    }
+
+    /**
+     * Report
+     */
+    function report(
+      node: AST.SvelteStartTag | AST.SvelteEndTag,
+      shouldHave: boolean,
+    ) {
+      const tagSrc = src.getText(node)
+      const match = /(\s*)\/?>$/m.exec(tagSrc)
```

Comment:
Probably don't need the m flag.

```suggestion
      const match = /(\s*)\/?>$/.exec(tagSrc)
```

I think the following HTML will make the wrong decision.

```html
<a attr="
  >
">
```